main.go

```diff
@@ -16,6 +16,7 @@ package main
 import (
 	"context"
 	"fmt"
+	"github.com/NYTimes/gziphandler"
```

Comment:
This import has to go into the block below (after line 32).

Reply:
ok